infrastructure/Taskfile.yml

```diff
@@ -844,7 +844,6 @@ tasks:
       dir: "{{.dir_env}}"
       cmds:
         - |
-          set -e
```

Comment:
Hvad dit ræsonement for ikke at ville stoppe ved en fejl?

Reply:
Vi så odd behavior hvor git succeedede men returnerede non-zero error code

Reply:
Men dette bliver alligevel irrelevant når jeg begynder at bruge taskfile loops